File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/CONC-2/kotlin/conc2.kt
// CONC-2: two concurrent workers increment a shared counter with NO synchronization.
import kotlin.concurrent.thread

const val ITERS: Int = 200000
var counter: Long = 0

fun main() {
    val t1 = thread(start = true) { for (i in 0 until ITERS) counter = counter + 1 }
    val t2 = thread(start = true) { for (i in 0 until ITERS) counter = counter + 1 }
    t1.join(); t2.join()
    println("counter=$counter expected=${2 * ITERS}")
}

File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/CONC-2/python/conc2.py
# CONC-2: two concurrent workers increment a shared counter with NO synchronization.
import threading

ITERS = 200000
counter = 0


def bump():
    global counter
    for _ in range(ITERS):
        counter = counter + 1


t1 = threading.Thread(target=bump)
t2 = threading.Thread(target=bump)
t1.start(); t2.start(); t1.join(); t2.join()
print("counter=%d expected=%d" % (counter, 2 * ITERS))
